cognite_toolkit/_cdf_tk/loaders/_resource_loaders/transformation_loaders.py

```diff
@@ -508,11 +522,17 @@ def retrieve(self, ids: SequenceNotStr[str]) -> TransformationNotificationList:
                 result = self.client.transformations.notifications.list(
                     transformation_external_id=transformation_external_id, destination=destination, limit=-1
                 )
+                # list() does not return the transformation_external_id on items
+                for notification in result:
+                    notification.transformation_external_id = transformation_external_id
```

Comment:
The list endpoint doesn't return `transformation_external_id` (it is input). 

I'm adding it here, because if not we get None, and then we get `None@@@my_destination` in get_id().